src/EditorFeatures/Core/LanguageServer/Handlers/Completion/CompletionResolveHandler.cs

```diff
@@ -28,7 +29,7 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
     /// See https://github.com/dotnet/roslyn/issues/55142
     /// </summary>
     [Method(LSP.Methods.TextDocumentCompletionResolveName)]
-    internal sealed class CompletionResolveHandler : IRequestHandler<LSP.CompletionItem, LSP.CompletionItem>
+    internal sealed class CompletionResolveHandler : ILspServiceRequestHandler<LSP.CompletionItem, LSP.CompletionItem>, ITextDocumentIdentifierHandler<LSP.CompletionItem, LSP.TextDocumentIdentifier?>
```

Comment:
That's because it's `TextDocumentIdentifier?` and not `TextDocumentIdentifier`. I wanted to avoid `ILspServiceDocumentRequestHandler` allowing null for TextDocumentIdentifier because it would usually be wrong.